syft/pkg/apk_metadata_test.go

```diff
@@ -105,8 +105,7 @@ func TestApkMetadata_FileOwner(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(strings.Join(test.expected, ","), func(t *testing.T) {
-			var i interface{}
-			i = test.metadata
+			var i interface{} = test.metadata
 			actual := i.(FileOwner).OwnedFiles()
```

Comment:
curious if there is a reason to not just do:
```suggestion
			actual := test.metadata.(FileOwner).OwnedFiles()
```

Reply:
it can actually be simpler: `actual := test.metadata.OwnedFiles()`